src/countBullsAndCows.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const { EXPECTED_NUMBER_LENGTH } = require('./constants');
+
+function countBullsAndCows(generatedNumber, userNumber) {
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < EXPECTED_NUMBER_LENGTH; i++) {
+    if (generatedNumber.includes(userNumber[i])) {
+      if (generatedNumber.indexOf(userNumber[i]) === i) {
+        bulls += 1;
+      } else {
+        cows += 1;
+      }
+    }
+  }
+
+  return [bulls, cows];
```

Comment:
Better to return `object` here
So order doesn't matter when you do destructuring